proto/Networking/Requests/Messages/GetUploadUrlMessage.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package POGOProtos.Networking.Requests.Social;
+package POGOProtos.Networking.Requests;
 
 message GetUploadUrlMessage {
```

Comment:
is it for GET_SIGNED_URL_FOR_PHOTO_UPLOAD?

Reply:
lv 40 make a stop

Reply:
the thing is, for a `GetUploadUrlMessage` to exist, we should have a `GET_UPLOAD_URL` in request type, but we don't. So this is not correct.

Reply:
true maybe is GET_SIGNED_URL_FOR_PHOTO_UPLOAD

Reply:
I'm guessing this is for `ADD_NEW_POI`

Reply:
maybe but not sur because poi is this https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Networking/Requests/Messages/PlatformClientActionsMessage.proto#L41

Reply:
or is a platform req style ptr8 // ref: Niantic.Platform.Protos.GetUploadUrlOutProto

Reply:
this is not ptr8

Reply:
je sais mais un autre peut etre... je chercher dans le dump GET_UPLOAD.. il n'y a rien mais pourtant il a bien se truc la bizzare...

Reply:
GET_SIGNED_GMAP_URL = 1105 ??? je me le demande....

Reply:
PING_ASYNC = 5027;                          // ???
	PING_DOWNSTREAM = 5030;                     // ???

peut etre l'un d'eux non ?

Reply:
There are some unused proto (that's why there are so much stuff you have I don't, I usually don't add ununsed stuff). So there may not be something to find.

Reply:
maybe, but devs use this in ipa pogo ++ ispoffer I say the protos this request I do not know ... see eliot if it has the mint of that ...

Reply:
ok i remove or add //TODO: ???

Reply:
REGISTER_PUSH_NOTIFICATION_SOCIAL_ACTION = 10101;
	UNREGISTER_PUSH_NOTIFICATION_SOCIAL_ACTION = 10102;
	OPT_OUT_PUSH_NOTIFICATION_CATEGORY_SOCIAL_ACTION = 10104;

this social actions need maybe added to RequestType resquet send the int32

https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Enums/SocialAction.proto#L21

Reply:
https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/83e1c437d2e19cdae4d8b1183c6e2cd37af34a5c

Reply:
reverted https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/535789e43539e79403083fef746a7292f18ffce2